packages/rancher-monitoring/generated-changes/overlay/templates/rancher-monitoring/clusterrole.yaml

```diff
@@ -90,4 +90,24 @@ rules:
   - 'get'
   - 'list'
   - 'watch'
+{{- if .Values.global.rbac.userRoles.createClusterExplorerRole }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
I added this as an aggregating ClusterRole (as opposed to a Role) to automatically provide this permission to members of the Project that contains Monitoring.